packages/plugin-meta-extractor/src/webpack-plugin.ts

```diff
@@ -0,0 +1,35 @@
+import path from 'node:path';
+import * as webpack from 'webpack';
+import { extractPluginMeta } from './meta-extractor.js';
+
+const DEFAULT_GENERATED_FILENAME = 'plugin.generated.json';
+
+export type GrafanaPluginMetaExtractorOptions = {
+  entryFile?: string;
+  outputAssetName?: string;
```

Comment:
Are we sure we want to expose this? To the best of my knowledge the name of the outputted file will be a contract of sorts with the Grafana server.

Reply:
So I guess the question is if we want plugin developers to be able to override it? If it lives in the build config it can be overriden by the plugin author.

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/805/commits/918a35cf4e548fedb6177a81265a4f61ee67f28b.